src/UniversalRewardsDistributor.sol

```diff
@@ -117,6 +116,7 @@ contract UniversalRewardsDistributor is IUniversalRewardsDistributor {
             MerkleProof.verifyCalldata(
                 proof,
                 rootOf[distributionId],
+                // TODO: why do we keccak256 it twice? we should highlight here
```

Comment:
can I get some explanation here?

Reply:
You can OZ merkle-tree's README especially [here](https://github.com/OpenZeppelin/merkle-tree#leaf-hash).

Reply:
So I actually removed the comment because there's no other way to build a valid leaf!